### ISSUE TITLE
Windows xlwings throwing Runtime Error code 53.

### ISSUE THREAD

**Issue by CLAV88** [xlwings/xlwings#604]
> Installed:
> 
> 1) auxlib
> 2) conda
> 3) xlwings
> 4) comtypes

> 
> Can't get xlwings to work correctly. Cannot call python from excel 64 bit running on a dual booted mac, currently installed on the Win64 bit partition.
> 
> Whenever I attempt to either run the macro for Importing Python UDF's via the addin or through the VBA developer ribbon just throws an error, tried uninstalling and reinstalling everything in the order listed above still doesn't work properly.
> 
> Should be noted that I am installing the packages through Pycharm running Python 3.5.
> 
> Any help would be appreciated.

**Comment by fzumstein** [comment]
you don't need auxlib but pywin32, but if you installed xlwings with conda, that shouldn't be the issue. Can you try to set the full path of the interpreter in the VBA settings and see if that helps?

**Comment by CLAV88** [comment]
Conda has a dependency on Auxlib, so I needed it for that.

I changed the settings in the VBA script to look at both the roaming and the local path, in two different attempts to get it to work, having read some of your other responses to similar issues (Error code 53), still no luck.

I've also tried manually pasting the dll's in the same working directory as python.exe and python.exe, that didn't work either.
